src/storm/solver/helper/ValueIterationOperator.h

```diff
@@ -291,11 +310,28 @@ class ValueIterationOperator {
                         OperandType const& operand, OffsetType const& offsets, uint64_t offsetIndex) const {
         STORM_LOG_ASSERT(*matrixColumnIt >= StartOfRowIndicator, "VI Operator in invalid state.");
         auto result{robustInitializeRowRes<RobustDirection>(operand, offsets, offsetIndex)};
-        AuxCompare<RobustDirection> compare;
+
         applyCache.robustOrder.clear();
 
+        if (applyCache.hasOnlyConstants.get(offsetIndex)) {
+            for (++matrixColumnIt; *matrixColumnIt < StartOfRowIndicator; ++matrixColumnIt, ++matrixValueIt) {
+                auto const lower = matrixValueIt->lower();
+                if constexpr (isPair<OperandType>::value) {
+                    STORM_LOG_THROW(false, storm::exceptions::NotImplementedException, "Value Iteration is not implemented with pairs and interval-models.");
+                    // Notice the unclear semantics here in terms of how to order things.
+                } else {
+                    result += operand[*matrixColumnIt] * lower;
+                }
+            }
+            return result;
+        }
+
+        // TODO I think this is a problem if we have probabilities and a state that is going to the vector, we don't count that
```

Comment:
TODO?

Reply:
I think we should just check these conditions before and make them an error because there's no way to fix it in valueiterationoperator.